### PR TITLE
Application to the Class of 2020

### PR DESCRIPTION
**Pull request by doomSDey**
# Application to the Class of 2020🎓

This pull request template helps you complete an application to the **Class of 2020**. Use the checklist below to verify you have followed the instructions correctly. 

## Checklist ✅

- [x] I have read the instructions on the README file before submitting my application. 
- [x] I made my submission by creating a folder on the `_data` folder and followed the naming convention mentioned in the instructions (`<username>`), added my profile picture and markdown file.
- [x] I have used the Markdown file template to add my information to the Year Book.
- [x] I understand that a reviewer will merge my pull request after examining it or ask for changes in case needed.
- [x] I understand I should not tag or add a reviewer to this Pull Request.
- [x] I understand the photo added to the template will be used in the ceremony "Graduate Walk". 
- [x] I have [added the event](http://www.google.com/calendar/event?action=TEMPLATE&dates=20200615T160000Z%2F20200615T183000Z&text=%24%20git%20remote%20%3Cgraduation%3E%20%F0%9F%8E%93&location=https%3A%2F%2Fwww.twitch.tv%2Fgithubeducation&details=) to my Calendar.

### CODE REVIEW

File: _data/doomSDey/doomSDey.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+name: Sudipta Dey # No longer than 28 characters
+institution: Sikkim Manipal Institute of Technology # no longer than 58 characters
+profile_pic: doomSDey.jpg # Name and extension of your profile picture(ex. mona.png) The picture must be squared and 544px on width and height.
```
Comment (Abhishekds94) [inline _data/doomSDey/doomSDey.md:4]: Hey @doomSDey,
Please change the following in your PR,
1. Looks like your image is of 1602x2314 px. Make your image a 1:1 size. (i.e, either 1602 x 1602 px or any other size)
2. Correct the image file name with the proper extension. In your case, it is to be `doomSDey.JPG` and not `doomSDey.jpg`

**Comment by doomSDey** [comment]
Done please check

**Comment by doomSDey** [comment]
Hi,
Thanks for all your warm wishes. I couldn't find a working link to the form hence I couldn't submit it. 
Thanks

**Comment by juanpflores** [comment]
It looks like your submission was done after we close shipping applications.

**Pull request merged**